pkg/agent/openflow/network_policy.go

```diff
@@ -723,8 +733,11 @@ func (c *client) InstallPolicyRuleFlows(ruleID uint32, rule *types.PolicyRule, n
 	// but the default drop flow is installed.
 	if nClause > 1 {
 		// Install action flows.
-		var actionFlows = []binding.Flow{
-			c.conjunctionActionFlow(ruleID, ruleTable.GetID(), dropTable.GetNext()),
+		var actionFlows []binding.Flow
+		if rule.Action != nil && *rule.Action == secv1alpha1.RuleActionDrop {
```

Comment:
considering we set the action always.. do we need this? i.e. rule.Action != nil? or we plan to revert Action to non pointer? i feel like i had discussed this with @tnqn  before my leave.. don't remember the decision ..

Reply:
Simply for precaution. Refactored.